### PR TITLE
Chapter 14

### PR DESCRIPTION
**Pull request by nuriman1**

### CODE REVIEW

**Comment by nuriman1** [comment]
ready

File: app/assets/stylesheets/relationships.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Relationships controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```
Comment (saobv-2223) [inline app/assets/stylesheets/relationships.scss]: remove file that never used

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -58,6 +58,20 @@ def destroy
     redirect_to users_url
   end
 
+  def following
+    @title = "Following"
+    @user = User.find_by params[:id]
+    @users = @user.following.paginate(page: params[:page])
```
Comment (saobv-2223) [inline app/controllers/users_controller.rb]: remove ()

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -58,6 +58,20 @@ def destroy
...
+    render "show_follow"
+  end
+
+  def followers
+    @title = "Followers"
+    @user = User.find(params[:id])
+    @users = @user.followers.paginate(page: params[:page])
```
Comment (saobv-2223) [inline app/controllers/users_controller.rb]: remove ()

File: app/helpers/relationships_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module RelationshipsHelper
+end
```
Comment (saobv-2223) [inline app/helpers/relationships_helper.rb]: remove file that never used

File: app/views/users/_follow_form.html.erb [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<% unless current_user?(@user) %>
```
Comment (saobv-2223) [inline app/views/users/_follow_form.html.erb]: remove ()

Comment (nuriman1) [reply app/views/users/_follow_form.html.erb]: done

File: app/controllers/users_controller.rb (hunk 3 above)
Comment (nuriman1) [reply app/controllers/users_controller.rb]: done

File: app/helpers/relationships_helper.rb (hunk 4 above)
Comment (nuriman1) [reply app/helpers/relationships_helper.rb]: done

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (nuriman1) [reply app/controllers/users_controller.rb]: done

File: app/assets/stylesheets/relationships.scss (hunk 1 above)
Comment (nuriman1) [reply app/assets/stylesheets/relationships.scss]: done

**Comment by nuriman1** [comment]
ready

**Comment by ngocvt-0484** [comment]
merge

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**